packages/bitcore-wallet-service/src/config.ts

```diff
@@ -73,6 +73,18 @@ module.exports = {
   maintenanceOpts: {
     maintenanceMode: false,
   },
+  simplex: {
+    sandbox: {
+      apiKey: 'simplex_sandbox_api_key_here',
```

Comment:
maybe better to keep this commented out. And is they are undef, make the code to ignore the related requests.

Reply:
Done